src/firewall-cmd.in

```diff
@@ -466,45 +466,29 @@ Direct Options
                        added [P]
 
 Lockdown Options
```

Comment:
Same for the CLI. We can parse them, but there is no reason to show them in the --help output.